WMF Framework/Configuration.swift

```diff
@@ -114,6 +115,17 @@ public class Configuration: NSObject {
         }
         return false
     }
+
+    @objc(mediawikiAPIURLForHost:)
+    public func mediawikiAPIURLForHost(_ host: String? = nil) -> URL? {
```

Comment:
should this live here or somewhere else?

Reply:
here is fine